### PR TITLE
Apellix CS landing page

### PR DESCRIPTION
**Pull request by caldav**
## Done

* New landing page at `/engage/industrial-drones-case-study`
- [x] [Copy doc](https://docs.google.com/document/d/1cRHdWw8d9OP71lwobO89eJtPENHH3TYzGuWrZgYcvz8)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Ensure [the page](https://www-ubuntu-com-canonical-websites-pr-4519.run.demo.haus/engage/industrial-drones-case-study) matches [the copy doc](https://docs.google.com/document/d/1cRHdWw8d9OP71lwobO89eJtPENHH3TYzGuWrZgYcvz8)
- Ensure the form works

### CODE REVIEW

File: templates/engage/industrial-drones-case-study.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{% extends "engage/base_engage.html" %}
+
+{% block meta_description %}This case study presents how Apellix, an industrial drone company, engineers safer work environments with Ubuntu.{% endblock %}
+
+{% block title %}Apellix engineers safer work environments with Ubuntu powered aerial robotics{% endblock %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1bqD9QKDj6ZfRkn-vbDrHHE_Zrk81cA5hFLRHZYFzzSA{% endblock meta_copydoc %}
```
Comment (anthonydillon) [inline templates/engage/industrial-drones-case-study.html]: This does not match the copy doc link

File: templates/engage/industrial-drones-case-study.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+{% block content %}
+
+{% include "engage/shared/_header.html" with title="Apellix engineers safer work environments with Ubuntu powered aerial robotics" image="5fd2e1aa-apellix_im3.png" url="#the-form" cta="Download the case study" %}
+
+
+<section class="p-strip is-shallow">
+  <div class="row">
+    <div class="col-7">
+            <div class="u-align--center col-12">
```
Comment (anthonydillon) [inline templates/engage/industrial-drones-case-study.html]: You dont need the `col-12` here

**Comment by anasereijo** [comment]
@caldav nice! Only suggestion is to hide the logo on small screens.

**Comment by caldav** [comment]
@anthonydillon not that I'm aware of. Thanks for catching these!
@anasereijo this is not an option in the template, but definitely something we should consider adding as a default behaviour! I'm going to A/B test on this one.

**Pull request merged**